support/releaseinfo/releaseinfo.go

```diff
@@ -69,14 +69,20 @@ type CoreOSImages struct {
 }
 
 type CoreRHCOSImage struct {
-	AzureDisk CoreAzureDisk `json:"azure-disk"`
+	AzureDisk     CoreAzureDisk     `json:"azure-disk"`
+	OpenStackDisk CoreOpenStackDisk `json:"openstack-disk"`
```

Comment:
Are you sure this is required? Doesn't seem to be used. What do you expect to go here?